sdk/include/opentelemetry/sdk/trace/simple_processor.h

```diff
@@ -33,7 +33,12 @@ class SimpleSpanProcessor : public SpanProcessor
    */
   explicit SimpleSpanProcessor(std::unique_ptr<SpanExporter> &&exporter,
                                bool is_export_async = false) noexcept
-      : exporter_(std::move(exporter)), is_export_async_(is_export_async)
+      : exporter_(std::move(exporter)),
+#ifdef ENABLE_ASYNC_EXPORT
```

Comment:
`is_export_async` will not be declared without `ENABLE_ASYNC_EXPORT` in `BatchSpanProcessorOptions`. I think we should keep the same behaviour with `SimpleSpanProcessor` and just remove `is_export_async_`.

Reply:
Done